### ISSUE TITLE
Add values for pH

### ISSUE THREAD

**Issue by dotps1** [Brewtarget/brewtarget#386]
Any chances in the future to add support for target pH values both for design and in the brewing logs?

**Comment by pricelessbrewing** [comment]
Worth noting that I added this to the BeerJSON recipe schema for mash steps, fermentation steps, and final beer pH in the recipe overview near the OG/FG/ABV section. 

https://github.com/beerjson/beerjson

**Comment by pricelessbrewing** [comment]
If this is added at any point, it should be clear that it's either the step pH (mash/fermentation), or final beer pH.

**Comment by matty0ung** [comment]
FYI we are actively working on BeerJSON support.  The nuts and bolts of reading the data in are coded, including support for a lot of weights and measures that we didn't have before.  What's going to take a little time (and which I have just started) is extending the data model to support all the things, such as pH, that are not present in BeerXML.

TLDR, this feature is coming...

**Comment by matty0ung** [comment]
I'm closing this on the grounds that BeerJSON is implemented (or rather will be once https://github.com/Brewtarget/brewtarget/pull/796 is merged).  IIRC we now have more pH fields than we used to.  We do not, yet, make much use of them, but I'd be happy to see new tickets with specific suggestions on that front...
